clippy_lints/src/lib.rs

```diff
@@ -344,6 +344,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry) {
     reg.register_late_lint_pass(box derive::Derive);
     reg.register_late_lint_pass(box types::CharLitAsU8);
     reg.register_late_lint_pass(box print::Pass);
+    reg.register_late_lint_pass(box write::Pass);
```

Comment:
I was hoping we could weave this into the `print::Pass`. Do you think that's feasible? Alternatively, would it be feasible to pull out the common code into functions that are called by both, but with arguments for the concrete names?

Reply:
Hi there!
I'm new to rust and of course this project. I'm not sure which is the right way to go about this, my plan was to implement the above lints then try to factor that logic into the existing print logic.
At the top level of the ast, there is not much in common with the print::Pass#check_expr so my inclination would be to factor common logic out, but I have no strong opinions on the subject.
Thanks for the pointer!

Reply:
On second thought, after doing some more work I think it should be easy putting this all together in print::Pass. Would we want to rename the module from print to something else then?

Reply:
That makes sense. Renaming the module is a good idea at that point. I think that just renaming it to `write` would be fine, because that's what `print!` refers to internally anyway. Better ideas for naming are very welcome!